src/pfs/pfs.proto

```diff
@@ -772,6 +772,13 @@ message ReposSummaryResponse {
   repeated ReposSummary summaries = 1; // repo summaries for the requested projects
 }
 
+message ForgetCommitRequest {
+  CommitPicker commit = 1;
+  bool dry_run = 2; // if true, this will report the metrics, but won't actually do the delete.
```

Comment:
You can omit the `dry_run` from this RPC. We'll implement it later once we have a way to better expose garbage collection information.